app/views.py

```diff
@@ -0,0 +1,29 @@
+from app import app
+from flask import render_template
+
+
+@app.route('/')
+def index():
+    return render_template('index.html')
+
+
+@app.route('/input/<input>')
+def print_user_input(input):
+	query = input.split('=')
+	variable = query[0]
+	value = query[1]
+	return u'Your query is {} which is equal to {}'.format(variable, value)

```

Comment:
please don't create an extra url for it — the whole idea is that ppl will (accidentally or out of curiosity) visit the *actual* webhook page — and that one has to show something.